common/src/main/java/earth/terrarium/hermes/utils/ElementParsingUtils.java

```diff
@@ -116,7 +117,48 @@ public static Alignment parseAlignment(Map<String, String> parameters, String ke
         return defaultValue;
     }
 
-    public static <A,B, C> @Nullable C parsePair(
+    public static Map<String, String> parseStyleAttribute(Map<String, String> parameters) {
+
+        // Allow for using 'style="..."' attribute values to compactly specify multiple attributes.
+        // These compact attributes may be in 'key:value' or boolean form.
+        // Booleans can be negated with a leading '!'; both forms may have short names or synonyms.
+        // Example: style="!obfuscated,b,em,color:lavender,bg:0 #4a38d0"
+
+        Map<String, String> result = new HashMap<>();
+        String[] specificers = parameters.get("style").split(",");
```

Comment:
This should be split on semicolon instead to match css more.

Reply:
Done with 2ac7410